src/views/InvolvementProfile/components/Membership/components/MemberList/components/MemberListItem/index.js

```diff
@@ -260,7 +256,7 @@ const MemberListItem = ({
         <Grid container alignItems="center" spacing={2}>
           <Grid item xs={1} style={rowStyle}>
             <Avatar
-              src={`data:image/jpg;base64,${avatar}`}
+              src={avatar ? `data:image/jpg;base64,${avatar}` : undefined}
```

Comment:
I don't think we're ever using the avatar without the image code at the front (`data:image/...`). Could we add this part to the `setAvatar(preferredImage || defaultImage);` line (82) so that it doesnt have to be defined 3+ times in the code?

Reply:
That's probably worth doing, but not really in the scope of this PR.